splice_machine/src/main/java/com/splicemachine/EngineDriver.java

```diff
@@ -96,7 +130,7 @@ public void close(SpliceSequence entity) throws Exception{
 
         /* Create a general purpose thread pool */
         final AtomicLong count = new AtomicLong(0);
-        this.threadPool = new ThreadPoolExecutor(0, config.getThreadPoolMaxSize(),
+        ThreadPoolExecutor tpe = new ThreadPoolExecutor(0, config.getThreadPoolMaxSize(),
```

Comment:
This is different from master for no reason (corePoolSize). Code below doesn't make sense if the number of core threads is 0.